01_01_Basic_Rules.txt

```diff
@@ -438,21 +439,86 @@ increase per level.
 Saving Throws
 ==============================
 
-	Saving throws are used in dire or critical situations to defend against very 
+	Saving Throws are used in dire or critical situations to defend against very 
 negative effects.
 
 - Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
 - Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
 - Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another)
 - Awareness: knowledge and understanding of surroundings (Ex: realizing someone has walked up behind you)
 
-The equations are as follows:
+The equations to calculate your bonuses to Saving Throws are as follows:
 
 - Will: 2*(Cha + Int - 6) Rational: Normally wisdom would be applied, so Will in Compound X is descried as your internal voice motivating you to endure a mental hardship.
 - Shock: 2*(Fort + Int - 6) Rational: You need to recognize that you are in shock, and resist its physical effects.
 - Reflex: 2*(Per + Dex - 6) Rational: In order to react quickly to an event you need to be able to see it coming.
 - Awareness: 2*(Per + Luc - 6) Rational: Line of sight and concentration are integral parts to being able to detect anomalies in the area.  
 
+==============================
+Feats
+==============================
+
+	As your character levels up and gets better at what they do, they develop
+certain talents and abilities that help them differentiate themselves from their
+peers. These special tallents and abilities are called "Feats". See the 
+"Leveling Up Rules" table to see how many feats you get at which levels. Unless 
+otherwise specified, you cannot take a feat more than once.
+
+	There are Five kinds of feats and we'll detail them below;
+
+* Passive Feats
+* Active Feats
+* Weapon Expertise Feats
+* Stat Feats
+* Class Feats
+
+Passive Feats:
+
+	When you take a feat, you may take a passive feat. Passive Feats are used 

```

Comment:
I'd nix the first sentence of Passive, Active and Weapon feats and make a note in leveling up about which feats can be chosen when. Or specify "When you gain a non-Class feat upon leveling up..."

Reply:
Second the removal of the first sentences.